.github/actions/templates/avm-setEnvironment/action.yml

```diff
@@ -45,7 +45,7 @@ runs:
 
         # Define PS modules to install on the runner
         $modules = @(
-            @{ Name = 'Az.Accounts' },
+            @{ Name = 'Az.Accounts'; Version = '2.19.0'},
```

Comment:
Is this hardcoded version required? Asking as I'm afraid this will stay here forever. 
I know we're doing the same below, still wondering 🙂

Reply:
We had it as the runner didn't have the latest but I just confirmed its updated and removed the pinned version :)